src/app/dashboard/dashboard.component.ts

```diff
@@ -92,13 +93,20 @@ export class DashboardComponent implements OnInit {
       && shelf.resourceIds.length === 0;
   }
 
-  getSurveys() {
-    this.submissionsService.getSubmissions(findDocuments({
-      'user.name': this.userService.get().name,
-      type: 'survey',
-       status: 'pending'
-    })).subscribe((surveys) => {
-      this.surveys = surveys;
+  getSurveysSubmissions() {
+    forkJoin([
+      this.submissionsService.getSubmissions(findDocuments({
+        'user.name': this.userService.get().name,
+        type: 'survey',
+        status: 'pending'
+      })),
+      this.submissionsService.getSubmissions(findDocuments({
+        type: 'exam',
+        status: 'complete'
+      }))
+    ]).subscribe(results => {

```

Comment:
When we know the result of a request it's better to use a descriptive name rather than a generic `result` (i.e. `surveys` and `exams`)